src/machine.rs

```diff
@@ -88,7 +92,7 @@ pub struct Evaluator<'tcx> {
     pub(crate) cmd_line: Option<Pointer<Tag>>,
 
     /// Last OS error.
```

Comment:
Now that this is a memory location, the comment should say e.g. how big it is (seems to always be 4 bytes?). Also, this was previously used for Windows stuff, `errno_location` is a Linux thing; can the two really share the same field?

Reply:
I'll add a comment about its size. It should work for windows in principle because windows uses an `u32` and linux uses an `i32`. 

I think only disadvantage of adding this is that we have the indirection of having to get the location pointer and then get the pointeé. But it shouldn't affect the capablities of setting and getting the error in either of the two platforms.